### PR TITLE
Update teaching_with_renkulab.rst

### PR DESCRIPTION
**Pull request by gavin-k-lee**
iFrame embeddings will be phased out in the next Renku release so we will point those who want to announce teachings directly to the JotForm.

### CODE REVIEW

File: docs/topic-guides/teaching_with_renkulab.rst [hunk 1]
```diff
@@ -31,8 +31,8 @@ We want to give your teachings the best Renku experience, whether they are one-o
 workshops or weekly lab sessions. This means ensuring adequate computational
 resources for you and your sessions, with a Renku admin on hand if there are
 problems with the platform. As such, we would very much appreciate if you could
-*announce* your teaching using the following form:
-https://renkulab.io/projects/team-renku/renku-teaching-announcements/files/blob/Form.ipynb.
+*announce* your teaching using the following form: `Announce your Renku teaching
```
Comment (rokroskar) [inline docs/topic-guides/teaching_with_renkulab.rst:34]: just wondering about the wording here... "announce your teaching" sounds a bit strange, what about "announce your course"?

Comment (rokroskar) [reply docs/topic-guides/teaching_with_renkulab.rst:34]: (I realize that's not what this PR is about but I didn't notice before) :)

Comment (gavin-k-lee) [reply docs/topic-guides/teaching_with_renkulab.rst:34]: Sure, we can do that.

**Review by rokroskar — APPROVED** [review]
Thanks!

**Pull request merged**